recipes/elektronn2/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "elektronn2" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "87484e4ef199144e0959a0574c4340d71800419ce99f24e106d04142bb1e9238" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/e/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not linux64]
```

Comment:
Thanks for merging my recipe. I will re-order those lines with the next update to the feedstock.